config/settings.yml

```diff
@@ -234,17 +232,6 @@ libraries:
     hours:
       library_slug: green
       location_slug: library-circulation
-  HOOVER:
-    label: Hoover Library
-    instructions: Items must be requested in advance and viewed on-site.
-    contact_info:
-      phone: "(650) 723-3563"
-      email: hooverarchives@stanford.edu
-    hold_pseudopatron: HOLD@HL
-    folio_hold_pseudopatron: 560fbbf8-35b6-4440-b218-6c4aa6625258
-    hours:
-      library_slug: hoover
-      location_slug: library-circulation
```

Comment:
are we sure that none of this gets used in the Aeon paging process?

Reply:
The links from Searchworks go directly to https://hoover.aeon.atlas-sys.com/